chrome/content/zotero/zoteroPane.js

```diff
@@ -1305,6 +1305,11 @@ var ZoteroPane = new function()
 						var type = mru ? mru.split(',')[0] : 'book';
 						await ZoteroPane.newItem(Zotero.ItemTypes.getID(type));
 						let itemBox = document.getElementById('zotero-editpane-item-box');
```

Comment:
By the way, do we think using something like `ZoteroPane.itemPane._itemDetails.getPane('info')` to replace the `getElementById` to access the inner elements is more readable?

(here's private property, but we can expose the `_itemDetails` too)

Reply:
`ZoteroPane.itemPane.getSection('info')`?

Reply:
The `ZoteroPane.itemPane` here includes `_messagePane`, `_duplicatesPane`, `_itemDetails`, and `_noteEditor`, which corresponds to the `N item in this view` message, duplicates pane, the container of item pane sections, and the note editor.

The `ItemPane` CE controls the status of these four parts, and the `ItemDetails` controls the status of the item pane sections.

Not sure if it makes sense/is necessary to make a shortcut of the `getSection` from `ItemDetails` in `ItemPane`?